pages/common/adb.md

```diff
@@ -17,3 +17,15 @@
 - Push an Android application to an emulator/device:
 
 `adb install -r {{path/to/file.apk}}`
+
+- Copy a file/folder from the target device:
+
+`adb pull {{path/to/file.ext}} {{destination/path}}`
```

Comment:
If it can be a file or a folder, the style we follow is `{{path/to/file_or_folder}}`. Also, {{destination/path}} should be {{path/to/device}}, as its actually the source in this case. And we can use the same token for the next command also.